### DISCUSSION TITLE
Diskussion der Ergebnisse

### DISCUSSION THREAD

**Discussion by rgb19** [rgb19/Gruppe5_WissArbeiten_WiSe_2324#11] [General]
Weitgehend für die Gruppe, die die Ergebnisse gemacht hat, aber weitere Ideen bitte anmerken!

**Comment by rgb19** [comment]
Signifikante Unterschiede für Alter und Ticketpreis; Überlebende in der Regel jünger und mit teurerem Ticket gereist. Frauen zahlten dabei im Mittel mehr für ihr Ticket, zudem sind "Sex" und "Survived" nicht gleichverteilt; der Unterschied stammt also (wahrscheinlich) nicht von einem anderen externen Faktor.

**Reply by Villkyrie** [reply]
interessant, wenn auch nicht überraschend.

**Comment by rgb19** [comment]
Unter den Passagieren bildeten die  1 Klasse die größte Gruppe Überlebender, und anders als bei den Verstorbenen sind die Klassen auch nicht stark verschieden groß. Frauen bilden immer die größere Gruppe Überlebender, dabei stärker verschieden als die ungleiche Verteilung von Männer und Frauen vermuten ließ. (341 Frauen zu 577 Männern)

**Reply by rgb19** [reply]
Das lässt sich auch gut in den ersten zwei Barplot-Grafiken erkennen

**Comment by rgb19** [comment]
Ergebnisse für den Zustiegshafen nicht überraschend für Cherboug stiegen in der Mehrheit Passage 1.Klasse dazu; es überlebten mehr und auch der größte Anteil an Frauen. Bei Southhampton kommt es ebenfalls zu keiner Überraschung, aber es ist zu beachten, dass dort der Großteil aller Passagiere einstieg.

**Comment by vafkoo** [comment]
Die Funktion categorial_stats berechnet verschiedene Statistiken für kategoriale Variablen. Dazu gehören der Modus, die empirische Entropie, das Phi-Streuungsmaß und die absoluten Häufigkeiten jeder Klasse. Für ordinale Merkmale werden zusätzlich das 0,25-, 0,5- und 0,75-Quantil berechnet.

Die Funktion MDStats berechnet den Durchschnitt und die Varianz einer metrischen Variable in Abhängigkeit von einer dichotomen Variable und führt einen T-Test mit diesen durch. [Oleksii Khudoliei]
